lib/live_content_report_exporter.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+
+class LiveContentReportExporter
+  attr_reader :publishing_apps
+
+  def initialize(publishing_apps)
+    @publishing_apps = publishing_apps
+  end
+
+  def total
+    editions.count
+  end
+
+  def file_path
+    @file_path ||= Rails.root.join("live_content_report_#{publishing_apps.join('_')}_#{Time.zone.now.strftime('%Y%m%d%H%M%S%N')}.csv")
```

Comment:
Might be clearer to extract the `publishing_apps.join('_')` and `Time.zone.now.strftime('%Y%m%d%H%M%S%N')` into variables and then interpolate them in.

Reply:
ugh, yeah, I should have spotted that (I used methods, not variables, because I prefer memoized methods to be oneliners).